### ISSUE TITLE
Request: method to update/edit movie/serie

### ISSUE THREAD

**Issue by marc0janssen** [vche/pycliarr#28]
Hi Vivien,

I would love to have a method for a movie and a serie to update/edit the existing datA in radarr/sonarr.

I need this to update tags in for a movie/serie.

Any help would be appreciated

Regards, Marco

**Comment by vche** [comment]
the api to edit an item is indeed missing, i will add it when i get time to work on pycliarr again.

i'm not sure which date you're referring to, e.g. it may not be possible to the date added as it is generated by sonarr/radarr. but all editable field will be editable.

you will be able to get the movie/serie, edit the returned item, and call the new update_movie/serie with the edited item to save it.

**Comment by marc0janssen** [comment]
SOrry was typo. I meant DATA instead of datE.

`you will be able to get the movie/serie, edit the returned item, and call the new update_movie/serie with the edited item to save it.`

That is exactly what I need!! 

- Get an Item
- Edit the returned item
- Put the edited Item back.

Thanks!!

**Comment by vche** [comment]
Fixed in v1.0.19
Note that some fields are not editable (like informations from scappers, those are not updated by sonarr/radarr).

From python:
```
from pycliarr.api import SonarrCli
sonarr_cli = SonarrCli('http://192.168.0.199:8989', '2ac2d0f66752xda3b41845e81dba5a84')

movie_info=sonarr_cli.get_serie(101)
movie_info.qualityProfileId = 3

updates_movie_info=sonarr_cli.edit_serie(movie_info)
```

From shell:
```
pycliarr -t "http://192.168.0.199:8989" -k "2ac2d0f66752xda3b41845e81dba5a84" sonarr get -i 101 -j >info.json
vim info.json # edit info
pycliarr -t "http://192.168.0.199:8989" -k "2ac2d0f66752xda3b41845e81dba5a84" sonarr edit -f info.json
```
